tests/test_standalone.py

```diff
@@ -72,3 +72,8 @@ def test_critics_consensus(happy_gilmore_content):
     assert standalone.critics_consensus("happy gilmore", happy_gilmore_content) == \
         "Those who enjoy Adam Sandler's schtick will find plenty to love in this gleefully juvenile take on professional golf; "+\
         "those who don't, however, will find it unfunny and forgettable."
+
+def test_num_of_reviews(happy_gilmore_content):
+    assert standalone.num_of_reviews("happy gilmore", happy_gilmore_content) == 55
```

Comment:
Same thing here... seems like this test will fail as soon as another review is added? 

Perhaps check that the value is (1) a valid integer _and greater_ than 55?